src/engraving/libmscore/line.cpp

```diff
@@ -1498,7 +1498,11 @@ bool SLine::setProperty(Pid id, const PropertyValue& v)
         _lineColor = v.value<mu::draw::Color>();
         break;
     case Pid::LINE_WIDTH:
-        _lineWidth = v.value<Millimetre>();
```

Comment:
There are similar conversions already in AbstractInspectorModel (valueToElementUnits and valueFromElementUnits). Maybe the bug is in one of them?

Reply:
No, this doesn't involve the Inspector. In fact, the Inspector works correctly; what's wrong is when you press Cmd+R to "reset" the element. That doesn't go through the Inspector.